<launch>
  <node name="data_visualizer" pkg="bwi_exp1_visualizer" type="bwi_exp1_visualizer" >
  <!-- <node name="data_visualizer" pkg="bwi_exp1_visualizer" type="bwi_exp1_visualizer" launch-prefix="gdb -ex run -args"> -->
    <param name="map_file" value="$(find bwi_exp1)/maps/exp1.yaml" />
    <param name="graph_file" value="$(find bwi_exp1)/maps/graph.yaml" />
    <param name="experiment_file" value="$(find bwi_exp1)/config/experiment.yaml" />
    <param name="data_directory" value="$(find bwi_exp1)/data-exp1-4-16-2013" />
    <param name="users_file" value="$(find bwi_exp1)/data-exp1-4-16-2013/users.yaml" />
  </node>
</launch>
